src/odemis/driver/test/simcam_test.py

```diff
@@ -23,8 +23,8 @@
 from odemis.acq.align.shift import MeasureShift
 from odemis.dataio import tiff
 from odemis.driver import simcam, simulated
-from odemis.util import timeout, test
-from odemis.util.test import assert_array_not_equal, assert_tuple_almost_equal
+from odemis.util import timeout, testing
+from odemis.util.testing import assert_array_not_equal, assert_tuple_almost_equal
```

Comment:
Can you update the code in this file that uses `assert_array_not_equal` and `assert_tuple_almost_equal` to use write it as `testing.assert_array_not_equal` and `testing.assert_tuple_almost_equal`. Then you can remove the line `from odemis.util.testing import assert_array_not_equal, assert_tuple_almost_equal`